Examples/NoModules/Chapter 10/Ex10_03.cpp

```diff
@@ -0,0 +1,23 @@
+// Using return type deduction with templates
+#include <iostream>
+#include <string>
+
+// Template for functions to return the larger of two values
+// Supports implicit converion of differently-typed arguments
+template <typename T1, typename T2>
+auto larger(T1 a, T2 b)
```

Comment:
my text was having const ref params, as in:
`auto larger(const T1& a, const T2& b)`
please verify

Reply:
In this case your text was still accurate. And the code thus not.